cmd/common/image/inspect.go

```diff
@@ -49,6 +53,21 @@ func NewInspectCmd() *cobra.Command {
 				return fmt.Errorf("inspecting image: %w", err)
 			}
 
+			ops := make([]operators.DataOperator, 0)
+			// empty for now, we'll progressively add operators
+		
+			gadgetCtx := gadgetcontext.New(
+				context.Background(),
+				image.String(),
+				gadgetcontext.WithDataOperators(ops...),
+				gadgetcontext.WithUseInstance(false),
+				)
+		
+			gadgetCtx.SetExtraInfo(true)
```

Comment:
I suggest making this an option so that we can do:

```go
			gadgetCtx := gadgetcontext.New(
				context.Background(),
				image.String(),
				gadgetcontext.WithDataOperators(ops...),
				gadgetcontext.WithGadgetExtraInfo(true),
		)
```

And then add only one new method to gadget-context:

```
func (c *GadgetContext) IncludeExtraInfo() bool
```

Similar to `useInstance` with `WithUseInstance` and `UseInstance`.